### PR TITLE
Fixed readInt_Test

### PR DESCRIPTION
**Pull request by AndriyVel05**

### CODE REVIEW

File: src/test/java/org/academy/codewars/ConsoleScannerTest.java [hunk 1]
```diff
@@ -20,26 +20,17 @@ public void beforeClass() {
...
+    public void testReadInt(String param, String input, int expectedValue, String expectedOutput) {
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+        ConsoleScanner consoleScanner = new ConsoleScanner(new Scanner(System.in));
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        PrintStream originalOut = System.out;
+        System.setOut(new PrintStream(outputStream));
+        int actualValue = consoleScanner.readInt(param);
+        System.setOut(originalOut);
+        assertEquals(actualValue, expectedValue);
+        assertEquals(outputStream.toString().trim(), expectedOutput.trim());
```
Comment (smelovd) [inline src/test/java/org/academy/codewars/ConsoleScannerTest.java:33]: remove trim() method

File: src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java [hunk 2]
```diff
@@ -88,4 +86,16 @@ public Iterator<Object[]> testReadIntArray() {
         return inputs.iterator();
     }
 
+    @DataProvider(name = "dpTestReadInt")
+    public Iterator<Object[]> dpTestReadInt1() {
+        List<Object[]> inputs = new ArrayList<>();
+        String param = "test int";
+        String lineSeparator = System.lineSeparator();
```
Comment (smelovd) [inline src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java:93]: remove variable

**Review by smelovd — CHANGES_REQUESTED** [review]

File: src/test/java/org/academy/codewars/ConsoleScannerTest.java [hunk 3]
```diff
@@ -20,26 +20,17 @@ public void beforeClass() {
...
+    public void testReadInt(String param, String input, int expectedValue, String expectedOutput) {
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+        ConsoleScanner consoleScanner = new ConsoleScanner(new Scanner(System.in));
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        PrintStream originalOut = System.out;
+        System.setOut(new PrintStream(outputStream));
+        int actualValue = consoleScanner.readInt(param);
+        System.setOut(originalOut);
+        assertEquals(actualValue, expectedValue);
+        assertEquals(outputStream.toString(), expectedOutput.replaceAll("\\n|\\r\\n", System.lineSeparator()));
```
Comment (smelovd) [inline src/test/java/org/academy/codewars/ConsoleScannerTest.java:33]: remove replace all

File: src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java [hunk 4]
```diff
@@ -117,4 +120,15 @@ public Iterator<Object[]> testReadIntArray() {
         return inputs.iterator();
     }
 
+    @DataProvider(name = "dpTestReadInt")
+    public Iterator<Object[]> dpTestReadInt1() {
+        List<Object[]> inputs = new ArrayList<>();
+        String param = "test int";
+        String expectedOutput1 = "Enter a " + param + " (int): ";
+        inputs.add(new Object[]{param, "25\n", 25, expectedOutput1});
+        String expectedOutput2 = "Enter a " + param + " (int): Invalid data format. An integer is expected.\nEnter a " + param + " (int): ";
```
Comment (smelovd) [inline src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java:129]: use " + System.lineseparator() + ", instead \n, move it after expectedOutput1

**Review by smelovd — CHANGES_REQUESTED** [review]

**Review by sbekberov — APPROVED** [review]

**Pull request merged**